trustable_cli/metrics.py

```diff
@@ -190,8 +197,10 @@ def _update_file_metrics(self, event):
             if not file["file"]:
                 continue
             # File type metrics
-            if re.search(FILE_TYPE_CODE, file["file"]):
+            if self.re_code_pattern.search(file["file"]):
```

Comment:
When a type of file is not found, the metric is not included. Code, binary and other should always been returned even when files of that type couldn't be found.